### PR TITLE
fix: fixes alt syntax for liatrio image in README.md again. :)

### PR DESCRIPTION
**Pull request by ianhundere**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,5 +89,5 @@ Use Docker to build and serve the content, but remember to rebuild the Docker im
 
 [Contributors](contributors.md)
 
-![](img/class_light.svg ':size=100x100 :class=light-mode-icon' alt="image of class; light mode")
-![](img/class_dark.svg ':size=100x100 :class=dark-mode-icon' alt="image of class; dark mode")
+![classroom image](img/class_light.svg ':size=100x100 :class=light-mode-icon :alt= classroom image; light mode')
+![classroom image](img/class_dark.svg ':size=100x100 :class=dark-mode-icon :alt= classroom image; dark mode')
```
Comment (jburns24) [inline README.md:93]:
```suggestion
![classroom image](img/class_light.svg ':size=100x100 :class=light-mode-icon :alt=classroom image; light mode')
![classroom image](img/class_dark.svg ':size=100x100 :class=dark-mode-icon :alt=classroom image; dark mode')
```

maybe a NP. not sure if spaces matter here

Comment (ianhundere) [reply README.md:93]: yah, that'll break it 😢

Comment (ianhundere) [reply README.md:93]: rather, you'll lose the word `classroom` for your definition

Comment (jburns24) [reply README.md:93]: Good to know thanks for the info

**Pull request merged**